src/main/java/org/opensearch/neuralsearch/highlight/SemanticHighlighterEngine.java

```diff
@@ -24,24 +25,18 @@
  * Engine class for semantic highlighting operations
  */
 @Log4j2
+@Builder
 public class SemanticHighlighterEngine {
     private static final String MODEL_ID_FIELD = "model_id";
-    private static final String DEFAULT_PRE_TAG = "<em>";
```

Comment:
where defaults tags are defined? You're removing current default ones, are they coming from core?

Reply:
Yes the default will anyway come from core if no customized tag provided by:
```
String[] preTags = fieldContext.field.fieldOptions().preTags();
String[] postTags = fieldContext.field.fieldOptions().postTags();
```